src/System.Management.Automation/namespaces/TransactedRegistrySecurity.cs

```diff
@@ -248,15 +248,15 @@ public override AccessRule AccessRuleFactory(IdentityReference identityReference
         /// <summary>
         /// <para>Creates a new audit rule, specifying the user the rule applies to, the access rights to audit, the inheritance and propagation of the
         /// rule, and the outcome that triggers the rule.</para>
+        /// </summary>
+        /// <returns>A TransactedRegistryAuditRule object representing the specified audit rule for the specified user, with the specified flags.
```

Comment:
The same.

Reply:
We're going to take a concerted approach to reorder all of the triple-slash comments as a separate approach. I've opened #18744 to track